Documentation/network/bgp-control-plane.rst

```diff
@@ -19,6 +19,21 @@ Currently a single flag in the ``Cilium Agent`` exists to turn on the
 
    --enable-bgp-control-plane=true
 
+Following values have to be set for helm installation
+
+.. code-block:: yaml
+
+   bgpControlPlane:
+     enabled: true
+
+.. note::
+
+   The BGP Control Plane feature is mutually exclusive with the :ref:`bgp`
+   feature and therefore it has to be disabled.
+   If both are enabled, the BGP will be prioritized and will probably fail.
+   In other words, this feature does not switch the BGP implementation
+   from MetalLB to GoBGP.
```

Comment:
This is a useful note, but is a bit confusing at times (it's hard to understand which BGP implementation we're talking about, and GoBGP/MetalLB aren't otherwise introduced in the page). Also if one has to be disabled anyway, I wouldn't try guessing what happens when both are enabled, let's just stick to “one has to be off”. Some suggestions below:

```suggestion
   The BGP Control Plane feature is mutually exclusive with the MetalLB-based :ref:`bgp`
   feature. To use the Control Plane, the older BGP feature has to be disabled.
   In other words, this feature does _not_ switch the BGP implementation
   from MetalLB to GoBGP.
```